docs/product/base/typography.html

```diff
@@ -299,6 +299,27 @@
             </div>
         </div>
     </div>
+    <p class="stacks-copy">Alternatively, you can use a vertical fade that will set <code>max-height</code> and sets a <code>vertical-mask</code>:</p>
+    <div class="stacks-preview">
+        {% highlight html %}
+<p class="v-truncate-fade-sm">…</p>
+<p class="v-truncate-fade-md">…</p>
```

Comment:
Do we make one of these the default and have sizing as a modifier?

`v-truncate-fade` is medium. `v-truncate-fade v-truncate-fade__sm` for small and `v-truncate-fade__lg`? Might be a bit more consistent with other components.